site/docs/relations/physical_relations.md

```diff
@@ -47,7 +47,45 @@ The nested loop join operator does a join by holding the entire right input and
 | Join Expression | A boolean condition that describes whether each record from the left set "match" the record from the right set. | Optional. Defaults to true (a Cartesian join). |
 | Join Type       | One of the join types defined in the Join operator.          | Required                                       |
 
-
+## Duplicate Eliminated Join Operator
+The Duplicate Eliminated Join, along with the [Duplicate Eliminated Get Operator](physical_relations.md#duplicate-eliminated-get-operator) are the two necessary operators that enable general subquery unnesting. (See the [Unnesting Arbitrary Queries](https://cs.emis.de/LNI/Proceedings/Proceedings241/383.pdf) paper for more information.)
+
+The Duplicate Eliminated Join is essentially a [Regular Join Operator](logical_relations.md#join-operator). It can have any regular join type, and its execution is the same. The main difference is that one of its children has, somewhere in its subtree, a dependency on the deduplicated result of the other. Therefore, this operator pushes the deduplicated result to its dependent child via the Duplicate Eliminated Get Operator. The side that will be deduplicated is specified in the Duplicate Eliminated Side property. The other side is the one that depends on the deduplication.
+
+| Signature            | Value                                                                                        |
+| -------------------- |----------------------------------------------------------------------------------------------|
+| Inputs               | 2                                                                                            |
+| Outputs              | 1                                                                                            |
```

Comment:
I think we need to say this has two outputs.  You can say it is "one output with a metadata back-channel" but I feel that is misleading.